build/constants.yaml

```diff
@@ -140,7 +140,7 @@
 - name: php-versions
   constants:
     php-runtime-base-tag: 20220803.3
-    php-fpm-runtime-base-tag: 20220803.2
+    php-fpm-runtime-base-tag: 202208017.1
```

Comment:
Should this be `20220817.1` (one less zero)?

Reply:
Thanks for catching it!